types/expectations.d.ts

```diff
@@ -0,0 +1,17 @@
+declare namespace Chai {
```

Comment:
is this used?

Reply:
hmm .. the merge up seems not to have deleted this lol